cranelift/foo.clif

```diff
@@ -0,0 +1,7 @@
+function u0:0(i64, i64) -> i64 {
```

Comment:
How is this file related to the rest of the PR?

Reply:
Oops it is not, just me being bad at git